src/main/java/com/minecolonies/entity/ai/citizen/fisherman/EntityAIWorkFisherman.java

```diff
@@ -461,6 +457,12 @@ private AIState doFishing()
         }
         if (caughtFish())
         {
+            if(worker.isFemale())
+            {
+                SoundUtils.playSoundAtCitizenWithChance(world, worker.getPosition(), FishermanSounds.Female.iGotOne, CHANCE_TO_PLAY_SOUND);
+            }
+            //todo add else for male sounds once ready.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)